### DISCUSSION TITLE
version.before_comma alternative in Formula

### DISCUSSION THREAD

**Discussion by laggardkernel** [Homebrew/discussions#1562] [Tap maintenance and Homebrew development] [answered]
`#{version.before_comma}` is only available in Cask but not Formula. Any alternative? Need some help.

What I've tried

- `version.split(",")`, returns `'undefined method 'split' for #<Version:0x00007feba0b75410>`

**Comment by carlocab** [comment, marked as answer]
Try `version.split(",").first`.

**Reply by laggardkernel** [reply]
@carlocab Nope, it doesn't work. I had already tried it before asked the question.

**Reply by carlocab** [reply]
oh right, sorry, not paying attention. Try `version.to_s.split(",").first`.
